src/java/org/apache/cassandra/gms/Gossiper.java

```diff
@@ -2023,10 +2018,13 @@ public List<String> reloadSeeds()
             return getSeeds();
         }
 
+        tmp.remove(getBroadcastAddressAndPort());
+
         // Add the new entries
         seeds.addAll(tmp);
         // Remove the old entries
         seeds.retainAll(tmp);
```

Comment:
so if you only define the self seed, the new logic tries to handle, but then this step here will make `seeds` empty... I don't really follow why we want to allow this behavior...

Reply:
@dcapwell isn't `seeds` empty when it is same in the descriptor? I am just following the behavior.